onc-healthit/inferno
# FI-465: Reorganize tests

- Move modules from `lib/app/modules` to `lib/modules`
- Move sequence tests from `test/sequence` to `lib/module/some_module/test`
- Move tests from `test/model` to `test/unit`
- Update `Rakefile` so `rake` runs tests in test directory and module directories
- Update `test_helper` so that fixture load path is determined dynamically

The code climate issues aren't really new, they just appear as new because the files were moved.

**Submitter:**
- [x] Internal ticket for this PR: https://oncprojectracking.healthit.gov/support/browse/FI-465
- [x] Internal ticket links to this PR
- [x] Internal ticket is properly labeled (Community/Program)
- [x] Internal ticket has a justification for its Community/Program label


**Reviewer 1:**

Name:
      where appropriate, and accomplishes the task's purpose

**Reviewer 2:**

Name:
      where appropriate, and accomplishes the task's purpose
